docs/dev/model_library_format.rst

```diff
@@ -0,0 +1,167 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+..    http://www.apache.org/licenses/LICENSE-2.0
+
+..  Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+Model Library Format
+====================
+
+About Model Library Format
+--------------------------
+
+TVM traditionally exports generated libraries as Dynamic Shared Objects
+(e.g. DLLs (Windows) or .so (linux)). Inference can be performed on those libraries by loading them
+into an executable using ``libtvm_runtime.so``. This process is very dependent on services provided
+by traditional OS.
+
+For deployment to unconventional platforms (e.g. those lacking traditional OS), the microTVM project
```

Comment:
Maybe use embedded instead?

```suggestion
For deployment to embedded platforms (e.g. those lacking traditional OS), the microTVM project
```

Reply:
i feel like it's not strictly limited to embedded though

Reply:
I agree.

to that end, I think we should word this as a general output format that produces what strictly TVM (code)generates. The creation of .so/.dll requires external toolchains called after tvm compilation -- such as LLVM and C compilers.
WDYT ?